metainsuranceorg.py

```diff
@@ -433,3 +442,34 @@ def process_newrisks_insurer(self, risks_per_categ, number_risks_categ, acceptab
 
         return not_accepted_risks
 
+
+    def market_permanency(self, time):     #This method determines whether an insurer or reinsurer stays in the market. If it has very few risks underwritten or too much cash left for TOO LONG it eventually leaves the market.
+                                                      # If it has very few risks underwritten it cannot balance the portfolio so it makes sense to leave the market.
+        cash_left_by_categ = np.asarray(self.cash_left_by_categ)
+
+        avg_cash_left = cash_left_by_categ.mean()
+
+        if self.cash < 100:         #If their level of cash is so low that they cannot underwrite anything they also leave the market.
```

Comment:
All these values used in this method (100, 4, 0.6, 24, 200) should be parameters. Even if we do not want to calibrate them, this would make that much easier to control and to understand.

Reply:
Done.